server/modules/user/service/userService.js

```diff
@@ -27,12 +30,16 @@ const inviteUser = async (user, surveyId, email, groupUuid) => {
   // If the survey is not published, only survey admins and system admins can be invited
   const survey = await SurveyManager.fetchSurveyById(surveyId)
   const surveyInfo = Survey.getSurveyInfo(survey)
+  const surveyLabel = Survey.getLabel(surveyInfo)
```

Comment:
if you don't pass the lang, then you will see the name. I think it's better to pass the lang in order to show the label if there is.

Reply:
Ok, added the lang